### PR TITLE
New tunnel cloaks

### PR DESCRIPTION
**Pull request by desvenlafaxine**
Updated sprites by 50N00b. 

image
image
image
image

### CODE REVIEW

**Comment by desvenlafaxine** [comment]
!review

**Comment by SilverTalismen** [comment]
Are these colorable like our current ones?

**Comment by desvenlafaxine** [comment]
> Are these colorable like our current ones?

There is a recolorable one, yes. The base sprite is white instead of dark grey so it's more faithful to the chosen color.

**Review by connorjg1 — APPROVED** [review]

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -625,4 +619,75 @@
...
+
+/obj/item/storage/backpack/cloak
+	name = "tunnel cloak"
+	desc = "A Vaurca cloak with storage pockets."
+	icon_state = "cape"
+	item_state = "cape"
+	w_class = ITEMSIZE_HUGE
+	slot_flags = SLOT_BACK
+	max_w_class = ITEMSIZE_NORMAL
+	max_storage_space = 28
```
Comment (Arrow768) [inline code/game/objects/items/weapons/storage/backpack.dm:634]: If you have the same variable value at the child and the parent you should not explicitly assign it at the child.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**